spec/vacuum_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# typed: false
+
+describe Redcord::VacuumHelper do
+  let!(:klass) do
+    Class.new(T::Struct) do
```

Comment:
i started this bad pattern in the all-in-one spec file lol. there's no reason for us to use dynamic class anymore.

```
class RedcordVacuumSpecModel < T::Struct
...
```